xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextGenerate.xtend

```diff
@@ -28,17 +30,17 @@ import org.xtext.gradle.tasks.internal.IncrementalXtextBuilderProvider
 class XtextGenerate extends DefaultTask {
 
 
-	@Accessors XtextSourceDirectorySet sources
+	@Accessors @Internal XtextSourceDirectorySet sources
 
-	@Accessors Set<Language> languages
+	@Accessors @Nested Set<Language> languages
 
-	@Accessors @InputFiles FileCollection xtextClasspath
+	@Accessors @Classpath FileCollection xtextClasspath
```

Comment:
looks like i cannot specify `@Classpath` and  `@Input(Files)` at the same time

Reply:
Yes, that would be redundant